### ISSUE TITLE
About prepared data

### ISSUE THREAD

**Issue by breeze0011** [SamsungLabs/DINAR#6]
Hello, I tried to put the prepared data's PKL files into Blender to display the mesh, but I didn't succeed. If possible, could you upload the prepared data's mesh results generated by SMPL-X? I wonder whether the results generated by SMPL-X myself are correct.

**Comment by david-svitov** [comment]
Hello! Could you please explain how exactly you are trying to open PKL files in Blender? PKL files do not contain meshes, but contain SMPL-X parameters. You can prepare your PKLs in a similar way to mine, for example by viewing their contents in Python using the Pickle library.
If you have questions about the compatibility of SMPL-X parameters, there is a discussion here: https://github.com/SamsungLabs/DINAR/issues/5

**Comment by breeze0011** [comment]
Thank you for your reply. There were some issues with the results generated by SMPL-X myself before, and I have found the error.

**Comment by breeze0011** [comment]
![image](https://github.com/SamsungLabs/DINAR/assets/130063990/da3f92df-13d2-4a8b-9a20-db8541e7e2f3)
I met this problem too. When you use smplx, you need to pay attention to the folder. You can try put only one image in this folder.
![image](https://github.com/SamsungLabs/DINAR/assets/130063990/18c8d8cc-c625-4b8c-a084-0e8481cf69f9)

**Comment by mayank64ce** [comment]
Yeah. What's happening is if I put any image from any other dataset along with snapshot people, it somehow morphs the snapshot outputs as well. So you are saying if I use only images from same dataset, I shouldn't have any problems ?

**Comment by breeze0011** [comment]
Sorry, I don't know why this situation occurs. I think the images in this folder will affect each other during the generation process.
